### ISSUE TITLE
Screen Reader is not announcing name for the close button present for available authorization dialog in windows.

### ISSUE THREAD

**Issue by msftedad** [swagger-api/swagger-ui#10320]
### Q&A (please complete the following information)
OS: Windows 11 24H2
Browser: Edge, chrome
Version: 133.0.3065.82 (Official build) (64-bit)
Method of installation: dist
Swagger-UI version: 5.17.14
Swagger/OpenAPI version: 3.0

### Content & configuration

Example Swagger/OpenAPI definition:
```yaml
# your YAML here
```

Swagger-UI configuration options:
```js
SwaggerUI({
  // your config options here
})
```

```
?yourQueryStringConfig
```

### Describe the bug you're encountering
Screen Reader is not announcing name for the close button present for available authorization dialog in windows. The lack of a name announcement for the close button in the available authorization dialog in Windows can greatly affect users who rely on screen readers, such as those with visual impairments. This issue makes it difficult for them to identify the button's function, resulting in confusion and challenges in navigating the dialog effectively.

### To reproduce...

1. Turn on Narrator.
2. Open the URL: [Swagger UI v5.17.14](https://codepen.io/formulahendry/pen/xxvQGaQ) in latest edge browser
3. Swagger Code pen home page will be opened.
4. Press tab key till reach Authorization button and press enter key to activate it.
5. Available authorizations dialog will be appeared.
6. Press tab key till reach close(x) button.
7. Verify whether screen reader is announcing name for close button or not.

### Expected behavior
Screen Reader should announce name for the close button present for available authorization dialog in windows. It should announce it as close button.

### Screenshots

https://github.com/user-attachments/assets/6178f595-6bee-4a85-b5f4-116813b38299

<img width="960" alt="Image" src="https://github.com/user-attachments/assets/f84e7a26-e84a-472a-aa12-929e1cc628e6" />
